src/main/java/org/broadinstitute/hellbender/tools/copynumber/FilterIntervals.java

```diff
@@ -281,51 +283,71 @@ private void validateFilesAndResolveIntervals() {
         inputReadCountFiles.forEach(IOUtils::canReadFile);
         Utils.validateArg(inputReadCountFiles.size() == new HashSet<>(inputReadCountFiles).size(),
                 "List of input read-count files cannot contain duplicates.");
-
-        //parse inputs to resolve intervals and validate annotated intervals, if provided
-        if (inputReadCountFiles.isEmpty()) {
-            //only annotated intervals provided (no counts)
+        
+        //parse inputs to resolve and intersect intervals
+        final LocatableMetadata metadata;
+        final List<SimpleInterval> resolved;
+        final List<SimpleInterval> intersected;
+        if (inputAnnotatedIntervalsFile != null && inputReadCountFiles.isEmpty()) {
+            //only annotated intervals provided
             annotatedIntervals = new AnnotatedIntervalCollection(inputAnnotatedIntervalsFile);
-            specifiedIntervals = new SimpleIntervalCollection(
-                    annotatedIntervals.getMetadata(),
-                    intervalArgumentCollection.getIntervals(annotatedIntervals.getMetadata().getSequenceDictionary()));
-            Utils.validateArg(specifiedIntervals.size() != 0, "At least one interval must be specified.");
-            Utils.validateArg(new HashSet<>(annotatedIntervals.getIntervals()).containsAll(specifiedIntervals.getIntervals()),
-                    "Annotated intervals do not contain all specified intervals.");
+            metadata = annotatedIntervals.getMetadata();
+            resolved = intervalArgumentCollection.getIntervals(metadata.getSequenceDictionary());
+            intersected = ListUtils.intersection(
+                    resolved,
+                    annotatedIntervals.getIntervals());
+        } else if (inputAnnotatedIntervalsFile == null && !inputReadCountFiles.isEmpty()) {
+            //only counts provided
+            final File firstReadCountFile = inputReadCountFiles.get(0);
+            final SimpleCountCollection firstReadCounts = SimpleCountCollection.read(firstReadCountFile);
+            metadata = firstReadCounts.getMetadata();
+            resolved = intervalArgumentCollection.getIntervals(metadata.getSequenceDictionary());
```

Comment:
Maybe check that the sequence dictionary is the same in counts file and intervals file?

Reply:
The intervals file may not have a sequence dictionary (e.g., we can just pass intervals as strings from the command line).  This code takes the sequence dictionary from the first read count file, which is used to check other inputs.  Probably we should check for the presence of a master dictionary, which may come from the intervals---let me try that.

Reply:
Hmm, actually it looks like checking interval files for a sequence dictionary would require a bit of extra code.  (Even `getBestAvailableSequenceDictionary` in `GATKTool` has a `TODO` for this.)  Will punt for the time being.